images/sbomer-generator/cyclonedx-init.gradle

```diff
@@ -0,0 +1,80 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2023 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+allprojects {
+  repositories {
+    maven {
+        url "http://${System.env.SBOMER_CACHE_SERVICE_HOST}:7070/maven/central/"
+        name "proxy-central"
+    }
+    mavenCentral()
+
+    maven {
+        url "http://${System.env.SBOMER_CACHE_SERVICE_HOST}:7070/maven/redhat-ga/"
+        name "proxy-redhat-ga"
+    }
+    maven {
+        url "https://maven.repository.redhat.com/ga/"
+        name "redhat-ga"
+    }
+
+    maven {
+        url "http://${System.env.SBOMER_CACHE_SERVICE_HOST}:7070/maven/indy-static/"
+        name "proxy-indy"
+    }
+    maven {
+        url "https://${System.env.SBOMER_INDY_HOST}/api/content/maven/group/static/"
+        name "indy"
+    }
+  }
+}
+
+initscript {
+  repositories {
+    maven {
+        url "https://plugins.gradle.org/m2/"
+    }
+  }
+
+  dependencies {
+    classpath "org.cyclonedx:cyclonedx-gradle-plugin:1.7.4"
```

Comment:
Are we able to control the version via the SBOMer config?

Reply:
I initially spent some time on this and failed, but will look again . Yes I wanted to make it configurable

Reply:
I'm fine for it being hardcoded in the first pass! I just saw the config mentioning the version and was curious how it works.